packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -584,6 +589,11 @@ const modalAnimatedScreens = (Navigator: typeof Stack) => (
       component={SelectCountry}
       options={SelectCountry.navigationOptions}
     />
+    <Navigator.Screen
+      name={Screens.SendConfirmationModal}
+      component={SendConfirmation}
+      options={sendConfirmationScreenNavOptions}
+    />
```

Comment:
Added this modal variation of the `SendConfirmation` screen so that it can be displayed nicely as a modal sheet on iOS, without breaking the existing flow with the normal `SendConfirmation` screen.

Reply:
yes it just displays as a normal modal there.